.github/workflows/leetcode_directory_writer.yml

```diff
@@ -0,0 +1,27 @@
+# The objective of this GitHub Action is to update the leetcode DIRECTORY.md file (if needed)
+# when doing a git push
+name: leetcode_directory_writer
+on:
+  push:
```

Comment:
Do we need this for every push? 🤔 I think it suffices to have this for a pull_request instead... What do you think @Panquesito7?